test/unit/index.test.ts

```diff
@@ -131,7 +131,15 @@ describe('api routes', (): void => {
       payload: MockConsentData.payload
     }
 
+    const expectedArgs = expect.objectContaining({
+      path: '/consents',
+      method: 'post',
+      payload: MockConsentData.payload
+    })
+
     const response = await server.inject(request)
+    expect(mockCreateConsent).toHaveBeenCalledTimes(1)
```

Comment:
we are testing here only that our handler has been called, but the handler itself is mocked, so the code of handler isn't tested here - do we have handlers tested elsewhere?

Reply:
I don't see that `CreateConsent` https://github.com/mojaloop/auth-service/blob/master/src/server/handlers/index.ts#L40
is implemented anywhere...

Reply:
> we are testing here only that our handler has been called, but the handler itself is mocked, so the code of handler isn't tested here - do we have handlers tested elsewhere?

Yes, the handlers are here: ` https://github.com/mojaloop/auth-service/tree/master/test/unit/server/handlers`